schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -1052,7 +1052,7 @@ Infiltration:
       AppG Used By TCDs: True
       AppG P_RMD Equals U_RMD: True
       AppG B_RMD Equals P_RMD: False
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: False
```

Comment:
Unsure why this would be different across rotations since it does not account for temp difference or wind speed.